.github/workflows/main.yml

```diff
@@ -274,7 +283,7 @@ jobs:
             ${{ env.IMAGE }}:${{ env.VERSION }}
```

Comment:
we don't push this anymore ?

Reply:
It's exactly the same behavior as before.

Reply:
right, my bad !